configs/mender_convert_config

```diff
@@ -227,9 +227,10 @@ source configs/mender_grub_config
 # You might want to override this to e.g provide state-scripts or providing
 # a private key to sign your artifact.
 mender_create_artifact() {
-  local -r device_type="${1}"
+  local -r device_types="${1}"
   local -r artifact_name="${2}"
   local -r image_name="${3}"
+  local -r all_device_types="$(for device_type in $device_types; do echo " --device-type $device_types"; done;)"
```

Comment:
Mmm. The changelog tag says that I can specify a comma separate list of device_types. Have you tried it? I think this line will only work with an space separated list...

Reply:
Ah, i see! I will fix it : )

Reply:
Also, from the ticket I see no mention of a comma-separated list. So is this really necessary?

If so, it should also have a documentation change to go with it.

But personally, I find this unnecessary, let's just stick with a space-separated list.